changelog/1.15.0.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "1.15.0"
+ådescription: "Released on 01/20/2021"
+---
+
+### Breaking Changes ❗
+
+- web: The root dashboard PWA manifest has been removed; uninstall the
+  Coder PWA. In lieu of this, Coder is moving forward with a PWA-per-application
+  approach. To install the Code Server PWA:
+  1. Log into Coder and select your environment.
+  2. Under **Applications** click **code** to launch code-server.
+  3. Follow your browser's instructions for installing the PWA.
+- web: Deprecated **Services** in favor of a declarative approach to creating
+  environments with YAML called **Environments as Code**. This feature is slated
+  to be released with Coder v2.0 (ETA: Q1 of 2021). For now, services will
+  continue to function as-is, though this feature will not receive any
+  improvements; they will be removed with Coder v2.0
+- web: The button to create an environment from an image tag has been removed
+
+### Features ✨
+
+- web: Added 30-day notices to dormant accounts listed in the **Manage** >
+  **Users** table; this warning indicates these accounts will be removed if not
+  reactivated
+- web: Added **Cluster Info** page to the Admin Panel. In the future, this page
+  will contain the administration and configuration of multiple clusters.
+- web: Redesigned Admin Panel
+  - Renamed the **General** page to **Infrastructure** and updated page with a
+    new design
+  - Redesigned the **Authentication**, **Appearance**, **Telemetry** and
+    **Dormancy** pages
+- web: Changed how Progressive Web Apps function; the code-server application
+  can now be installed as a PWA.
+- web: Added tooltip with extended information that shows when hovering over the
+  Image info in the summary card on the `Environments` page
+- web: Redesigned the Coder Dashboard to use NextJS and SSR. Page loads should
+  be quicker ⚡.
```

Comment:
Cluster info tab in admin page landed in this release so we might want to mention it here.

Reply:
We did - look on line 26